data/human/human missions.txt

```diff
@@ -3996,3 +3996,129 @@ mission "Fumbling Mugger"
+			`	You hold Riley as she silently weeps. Time goes by, but nobody pays it any mind.`
+			label close
+			`	You take a seat, feeling the emotion of the moment wash over you as Riley silently weeps. Minutes go by, unnoticed.`
+			`	Without warning, you hear Church's voice from a corner of the room. It's coming from some sort of computer system, held aloft by hundreds of cables dangling from the ceiling.`
+			`	"It... worked." He sounds shocked. Riley's head whips around, and you reflexively move your hand toward your holstered gun. "Please, don't be alarmed. I've been working on a personal project for over a decade now. My mind was uploaded to the modified Core you're looking at. I had to say goodbye - I couldn't know if it would work. You can't upload a mind while it's still alive. I tried that many times."`
+			choice
+				`	(Stay silent)."`
+					goto continue
+				`	"Is this... legal?"`
+			`	"Well, no," he replies. "Not under Deep law. I assure you that my only intention is to continue my work; nothing more."`
+			`	A quick search shows a 30000 credit reward for turning in a mind upload.`
+			label continue
+			`	"Riley, what do you think?"`
+			`	"Dad, is that really you? How do you feel?"`
+			`	She steps closer to the Core, and they talk for a few tentative minutes. As far as you can tell, the mind that she is talking to is indistinguishable from the man you came to meet.`
+			`	Finally, she turns to you. "It really is my dad. It's a lot to get used to, but... how many people get a second chance to spend time with their father before he's gone? Thank you for bringing me out here, and for coming. I'll never forget your part in this. For now, I'd like to stay here. You go - I'll find my way back."`
+			choice
+				`	"Are you sure? There's enough room for all of us on my ship. You can stay for as long as you want."`
+					goto suggest
+				`	(Say goodbye).`
+					goto goodbye
+				`	(Turn the Core in for a bounty.)`
+					goto bounty
+				`	(Draw your gun and destroy the Core).`
+			`	You advance on the Core and start firing at it. Church begs for you to stop until his voice cuts off. Lights on the Core turn from white to red, then fade. Riley hasn't moved, stunned.`
+			`	You head back to your shuttle, leaving behind a father survived by his daughter; no more and no less than nature intended.`
+			label suggest
+			`	"It's a nice idea," she smiles, "but I'm not a starfarer. The big city is where I'll end back up, eventually."`
+			label goodbye
+			`	Respecting Riley's request to stay with her father, you make your farewells and head for the door.`
+			`	You can feel her smile on your back until the door closes behind you. You catch yourself smiling as well, as you make your way back to your shuttle and return to the stars.`
+				decline
+			label bounty
+			action
+				payment 30000
+			`	You say your goodbyes and leave. From across the street, you make a call to the local Deep government office, and watch as they descend on the house.`
```

Comment:
As I recall, Pilot is around the Paradise Worlds? Isn't it a test site for Lovelace Labs? The Deep aren't particularly nearby.